src/lib.rs

```diff
@@ -155,6 +157,26 @@ impl Drop for FfiHandle {
     }
 }
 
+/// This function should be called to enable logging to a file
+#[no_mangle]
+pub extern "C" fn init_logging() -> int32_t {
+    env::set_var("RUST_LOG", "trace");
```

Comment:
this can probably be made `env::set_var("RUST_LOG=crust,routing,safe_", "trace");` else we're looking at a lot of messages from third party libs getting logged too with trace set